app/models/concerns/authorization_extensions/volumetrie.rb

```diff
@@ -0,0 +1,30 @@
+module AuthorizationExtensions::Volumetrie
+  extend ActiveSupport::Concern
+
+  included do
+    add_attribute :volumetrie_appels_par_minute
+    add_attribute :volumetrie_justification
+
+    validates :volumetrie_appels_par_minute,
+      presence: true,
+      if: -> { need_complete_validation?(:volumetrie) }
+
+    validates :volumetrie_justification,
+      presence: true,
+      if: -> { need_complete_validation?(:volumetrie) && volumetrie_is_not_the_smallest }
+  end
+
+  def volumetrie_is_not_the_smallest
+    volumetrie_appels_par_minute.to_i > default_volumetrie
+  end
+
+  def default_volumetrie
```

Comment:
```suggestion
  def smallest_volumetrie
```

C'est peut-être le default dans la vue, mais d'un point de vue modèle ça n'est pas le cas, ni évident, et c'est brainfuck